gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
same here.

Reply:
gradle wrapper files seem to be necessary & it isn't recommended to add those to .gitignore